docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/DockerComposeConverter.java

```diff
@@ -61,10 +61,19 @@ public CubeContainers convert() {
         CubeContainers cubeContainers = new CubeContainers();
 
         Set<String> names = dockerComposeDefinitionMap.keySet();
-
-        for(String name : names) {
-            CubeContainer cubeContainer = convertContainer(asMap(dockerComposeDefinitionMap, name));
-            cubeContainers.add(name, cubeContainer);
+        boolean isV2 = names.contains("version") && dockerComposeDefinitionMap.get("version").equals("2") && names.contains("services");
+        if (isV2) {
+            Map<String, Object> dockerComposeV2DefinitionMap =  asMap(dockerComposeDefinitionMap, "services");
+            Set<String> services = dockerComposeV2DefinitionMap.keySet();
+            for(String name : services) {
+                CubeContainer cubeContainer = convertContainer(asMap(dockerComposeV2DefinitionMap, name));
```

Comment:
I think we should need to create a new factory for docker compose 2 changes, notice that there are a lot of things we should adapt. For example: 
[ ] https://docs.docker.com/compose/compose-file/#build
[ ] adding network/volumes
[ ] in docker-compose v2 if no network is provided one is created by default
[ ] you can use build and image at the same time
[ ] we need to implement label support

Info at: https://github.com/docker/compose/releases/tag/1.6.0
Info about networking: https://docs.docker.com/compose/networking/